### PR TITLE
add style.css and style the buttuns

### PR DESCRIPTION
**Pull request by meller-rivka**

### CODE REVIEW

**Comment by ChayaMantel** [comment]
i think that it is ok
good luck!

File: Git.html [hunk 1]
```diff
@@ -5,7 +5,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
 </head>
-<body style="background-color: rgb(38, 0, 255);">
```
Comment (ChayaMantel) [inline Git.html:8]: a very nice color

**Review by ChayaMantel — COMMENTED** [review]
i think that it is ok

**Review by ChayaMantel — COMMENTED** [review]
ok

**Review by meller-rivka — COMMENTED** [review]
good!

**Review by ChayaMantel — COMMENTED** [review]
ok

**Pull request merged**